sphinxext/opengraph/event_handler.py

```diff
@@ -0,0 +1,46 @@
+from typing import Dict, Any
+from sphinx.application import Sphinx
+from docutils import nodes
+from urllib.parse import urljoin
+from .util import sanitize_title, make_tag, add_tag_from_config
+from .visitor import OpenGraphVisitor
+
+
+def html_page_context(
+    app: Sphinx,
+    pagename: str,
+    templatename: str,
+    context: Dict[str, Any],
+    doctree: nodes.document,
+) -> None:
+    if not doctree:
+        return
+    # Set length of description
+    try:
+        desc_len = int(app.config["ogp_description_length"])
+    except ValueError:
+        desc_len = 200
```

Comment:
Constants shouldn't be defined twice. 
Since `app.add_config_value("ogp_description_length", 200, "html")` is already in `__init__.py`, this isn't necessary.